main.c

```diff
@@ -382,7 +382,7 @@ int main(int argc, char** argv) {
       break;
 
     default:
-      printf("ARGC: %i\n", argc);
+      printf("Unknown argument at index %i\n", optind);
```

Comment:
Can we print the unrecognized argument string instead?

Reply:
We had a quick discussion about this in our review and without further investigation into the workings of getopt here, this was the least intrusive patch we could find. We can take another look at this to see if the needed string is readily available.

Reply:
`argv[optind]` should work.

Reply:
We were unsure re this, because of folding of short options.